exchange/adapter_builders.go

```diff
@@ -334,5 +335,6 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderYieldmo:           yieldmo.Builder,
 		openrtb_ext.BidderYieldone:          yieldone.Builder,
 		openrtb_ext.BidderZeroClickFraud:    zeroclickfraud.Builder,
+		openrtb_ext.BidderUndertone:         undertone.Builder,
```

Comment:
Preference here is to add builder in ascending order. So should move `openrtb_ext.BidderUndertone: undertone.Builder,` to line 322